models/llm_openai/model.py

```
@@ -0,0 +1,74 @@
+        inference_results = [self._run_prompt(text) for text in texts]
+        moral_outrage_list = []
+        timestamp = get_current_timestamp()
+        try:
+            for inference, text in zip(inference_results, texts, strict=True):
+                moral_outrage_list.append(MoralOutrage(
+                    text_id=str(uuid4()),
+                    text=text,
+                    moral_outrage_score=int(inference),
+                    label_timestamp=timestamp,
+                ))   
+        except:
```

Ruff (0.15.9)

[error] Do not use bare `except`

(E722)